crypto/evp/evp.h

```diff
@@ -1491,6 +1491,7 @@ void ERR_load_EVP_strings(void);
 # define EVP_F_EVP_DECRYPTFINAL_EX                        101
 # define EVP_F_EVP_DIGESTINIT_EX                          128
 # define EVP_F_EVP_ENCRYPTFINAL_EX                        127
+# define EVP_F_EVP_ENCRYPTUPDATE                          180
```

Comment:
master has 219, but that's not a problem, right?

Reply:
Nope.  We haven't retained ABI compatibility between 1.0.x and 1.1.0, so the number differences are not significant.

Reply:
> Nope.

Ok, I take that for a "yes" ;-).

A friend of mine once said: "Never ask a negated question. Because when you get a 'no', you won't know what the other one means". The same friend never answers negated questions with "yes/no", he always uses "correct/incorrect". (Needless to say that this friend is a mathematician...)

Reply:
Careful... I'm playful with that at times.

You're interpretation is correct, though...